tests/test_gsy_core/test_redis_area_market_communicator.py

```diff
@@ -1,41 +1,45 @@
+# pylint: disable=redefined-outer-name
 from unittest.mock import patch, Mock
 
 import pytest
 from redis import Redis
+from gsy_framework.redis_channels import AggregatorChannels
+
 
 from gsy_e.gsy_e_core.redis_connections.aggregator import AggregatorHandler
 from gsy_e.gsy_e_core.redis_connections.area_market import ExternalConnectionCommunicator
 
 
-@pytest.fixture(scope="function", autouse=True)
-def strict_redis():
+@pytest.fixture(scope="function", autouse=True, name="strict_redis")
```

Comment:
I think that name was not mandatory here, because these fixtures are not used as function arguments as far as I see, but rather applied for all test functions of the module. Does not change the functionality though as far as I recall

Reply:
You are right. With `autouse=True` the name is not needed. At least in our tests. Let me remove it.